src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
+    return function inner(b) {
```

Comment:
There is no need to create an inner function. getSum should directly take two arguments and return the result or error based on the condition.